mvc/migration/migratingfrommvc5.rst

```diff
@@ -1,257 +1,244 @@
-	{
-		app.UseMvc(routes =>
-		{
-			routes.MapRoute(
-				name: "default",
-				template: "{controller=Home}/{action=Index}/{id?}");
-		});
-	}
+``UseStaticFiles`` adds the static file handlers. As mentioned previously, the ASP.NET runtime is modular, and you must explicitly opt in to serve static files. For more information, see `Application Startup <http://docs.asp.net/en/latest/fundamentals/startup.html>`_ and `Routing <https://docs.asp.net/en/latest/fundamentals/routing.html>`__.
 
-At this point we are ready to create a simple Controller and View. Add a Controllers folder and a Views folder to the project. Add an MVC Controller called HomeController.cs class to the Controllers folder and a Home folder in the Views folder. Finally, add an Index.cshtml MVC View Page to the Views/Home folder. The project structure should be as shown:
+Add a controller and view
+========================================
+
+In this section you'll add a minimal controller and view to serve as place holders for the MVC 5 controller and views you'll migrate in the next section.
+
+- Add a *Controllers* folder.
+- Add an MVC controller class with the name *HomeController.cs* to the *Controllers* folder.
+- Add a *Views* folder. 
+- Add and a *Views/Home* folder.
+- Add an *Index.cshtml* MVC view page to the *Views/Home* folder. 
+
+The project structure is shown below:
 
 .. image:: migratingfrommvc5/_static/project-structure-controller-view.png
 
-Modify Index.cshtml to show a welcome message:
+Replace the contents of the *Views/Home/Index.cshtml* file with the following:
 
 .. code-block:: html
 
 	<h1>Hello world!</h1>
 
-Run the application - you should see Hello World output in your browser.
+Run the app.
 
 .. image:: migratingfrommvc5/_static/hello-world.png
 
 .. _migrate-controllers-views-static:
 
-Migrate Basic Controllers, Views, and Static Content
-----------------------------------------------------
+See :doc:`../controllers/index` and :doc:`../views/index` for more information.
+
+Now that we have a minimal working MVC 6 project, we can start migrating functionality from the MVC 5 project. We will need to move the following:
+
+- client-side content (CSS, fonts, and scripts)
+- controllers
+- views
+- models
+- bundling 
+- filters
+- Log in/out, identity (This will be done in the next tutorial.)
+
+Controllers and views
+========================================
+
+- Copy each of the methods from the MVC 5 ``HomeController`` to the MVC 6 ``HomeController``. Note that in MVC 5, the built in template generated return type is `ActionResult <https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/ActionResult.cs>`_, in MVC 6 the templates generate `IActionResult <https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Abstractions/IActionResult.cs>`__ methods. ``ActionResult`` is the only implementation of ``IActionResult``, so there is no need to change the return type of your action methods.
```

Comment:
You are starting the "Controllers and views" section without an intro sentence. Add a sentence to tell the reader what you are explaining here.